contracts/src/v0.8/dev/shared/interfaces/OwnableInterface.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
```

Comment:
It appears no pragma has been chosen, I'd recommend 0.8.19 and to use the named mappings

Reply:
done

Reply:
Fully my bad, but interfaces should be `^0.8.0` and contracts `0.8.19` (note no `^` here) and I commented this on an interface 😅